File: LeetCode_Marathons/LCM6/Easy/Range_Addition_II/a.go
func maxCount(m int, n int, ops [][]int) int {
    minX, minY := m, n;

    for _, opInfo := range ops {
        minX = min(minX, opInfo[0])
        minY = min(minY, opInfo[1])
    }

    return minX * minY
}

func min(a, b int) int {
    if a < b {
        return a
    }
    return b
}

File: LeetCode_Marathons/LCM6/Easy/Range_Addition_II/a.py
class Solution:
    def maxCount(self, m: int, n: int, ops: List[List[int]]) -> int:
        minX: int = m
        minY: int = n

        for opInfo in ops:
            minX = min(minX, opInfo[0])
            minY = min(minY, opInfo[1])
        
        return minX * minY